.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -7,23 +7,16 @@ body:
     id: currentproblem
     attributes:
       label: Problem
-      description: Tell us about the current problem that you face that this feature might help you with.
+      description: Tell us about the problem.
     validations:
       required: true
   - type: textarea
-    id: suggestedfeat
+    id: possiblesolution
     attributes:
-      label: Suggested feature
-      description: Tell us about the feature that might solve the above problem.
+      label: Possible Solution
+      description: If you have an idea, please tell us what might solve the above problem.
     validations:
       required: true
-  - type: textarea
-    id: why
-    attributes:
-      label: Why should we have this feature?
-      description: List any extra benefits, other than solving your problem, that this feature will bring to the repository. If none, leave blank.
```

Comment:
I think we should keep this, it is useful to know more information. 🙂

Reply:
It sounds good in theory but usually, people use the above two boxes to explain their problems already. I myself found this box unnecessary since to explain my feature, I would use the above two boxes.